src/controller/eme-controller.js

```diff
@@ -545,12 +546,18 @@ class EMEController extends EventHandler {
     let keyMessageXml;
 
     switch (keysListItem.mediaKeySystemDomain) {
+    // eslint-disable-next-line no-case-declarations
     case KeySystems.PLAYREADY:
+      const utfEncodingType = this._drmSystemOptions.utfEncodingType;
+      const isUtf8EncodedPlatform = utfEncodingType === 'utf-8';
+      const uIntArray = isUtf8EncodedPlatform ? new Uint8Array(keyMessage) : new Uint16Array(keyMessage);
```

Comment:
ok, so if we don't set `utfEncodingType`, it's still the same as before, which means we don't need to add this option for other platforms using Hls.js.

Reply:
Yes thats correct.